modules/contact-form/grunion-contact-form.php

```diff
@@ -1838,12 +1838,15 @@ function __construct( $attributes, $content = null ) {
 		self::$current_form = $this;
 
 		$this->defaults = array(
-			'to'                 => $default_to,
-			'subject'            => $default_subject,
-			'show_subject'       => 'no', // only used in back-compat mode
-			'widget'             => 0,    // Not exposed to the user. Works with Grunion_Contact_Form_Plugin::widget_atts()
-			'id'                 => null, // Not exposed to the user. Set above.
-			'submit_button_text' => __( 'Submit', 'jetpack' ),
+			'to'                     => $default_to,
+			'subject'                => $default_subject,
+			'show_subject'           => 'no', // only used in back-compat mode
+			'widget'                 => 0,    // Not exposed to the user. Works with Grunion_Contact_Form_Plugin::widget_atts()
+			'id'                     => null, // Not exposed to the user. Set above.
+			'submit_button_text'     => __( 'Submit', 'jetpack' ),
+			'customThankyou'         => '', // Whether to show a custom thank you response after submitting a form. '' for no, 'message' for a custom message, 'redirect' to redirect to a new URL.
+			'customThankyouMessage'  => __( 'Thank you for your submission!', 'jetpack' ), // The message to show when customThankyou is set to 'message'.
+			'customThankyouRedirect' => '', // The URL to redirect to when customThankyou is set to 'redirect'.
```

Comment:
Note that there's a mixture of underscore and camelcase attributes here. It's tricky because of how values are passed across a mixture of JS and PHP which have different standards. :-/

Reply:
Just noticed you commented here why; https://github.com/Automattic/jetpack/pull/13745/files#r334693355

Might be worth leaving a code comment here so as to not confused future developers?

Reply:
👍🏻